Normalizing paths before checking equality

talesbarreto opened the pull request:
Closes https://github.com/talesbarreto/pull_request_coverage/issues/24

mahmuttaskiran commented:
This did not solve the problem when I tried with the same example in #24. I wrote a unit test to explain the situation:

```
const _content3 = '''
SF:./lib/src/extensions/string.dart
DA:4,1
DA:5,1
end_of_record

''';

test("should normalize paths and return lines that are not covered", () {
    final useCase = GetUncoveredFileLines();
    final result = useCase(_content3.split("\n"), "/lib/src/extensions/string.dart");

    expect(result != null);
});
```

There, we assume the path we get from `git diff` command is `/lib/src/extensions/string.dart` and the path we have in the lcov.info file is `./lib/src/extensions/string.dart`. Normalizing these paths didn't work. 

Btw, I'm not sure that this tool needs to handle this issue. Normally, the lcov.info file will have the path without the current directory prefix. This issue happens because we are manipulating the lcov.info content. Maybe we need to edit the logic in our scripts that run the test suites.

talesbarreto commented:
Oh, I see!

Thank you for write this test. In this particular case, I think the output is correct because `/lib/src/extensions/string.dart` is an absolute path and `./lib/src/extensions/string.dart` is a relative path, which means they may indeed refer to different files.

I added a `.` to make both paths relative, and the test failed as well. So I changed `GetUncoveredFileLines` to normalize both paths before compare them. 

I also believe that `coverage` will not generate paths with `./` prefix but I see no problem to handle this case. I pushed a commit fixing the first solution and added your test as well (with the additional `.`)
